src/Microsoft.DotNet.Build.Tasks/VersionTools/BaseDependenciesTask.cs

```diff
@@ -16,6 +16,16 @@ namespace Microsoft.DotNet.Build.Tasks.VersionTools
 {
     public abstract class BaseDependenciesTask : Task
     {
+        protected static string RawUrlMetadataName = "RawUrl";
```

Comment:
Fixed `static` -> `const`, and just to make sure I have some ok reasoning, `internal` rather than `protected` to make it easier for others in the assembly to reuse but also limit potential for someone to depend on it who shouldn't?